pom.xml

```diff
@@ -206,6 +206,26 @@ under the License.
       <artifactId>guava</artifactId>
       <version>23.0</version>
     </dependency>
+    <!-- Jackson Dependencies -->
```

Comment:
Does the app depend directly on Jackson? If not, then you shouldn't need these dependencies.